pavelib/quality.py

```diff
@@ -92,7 +85,8 @@ def find_fixme(options):
 @cmdopts([
     ("system=", "s", "System to act on"),
     ("errors", "e", "Check for errors only"),
-    ("limit=", "l", "limit for number of acceptable violations"),
+    ("lower_limit=", "l", "lower limit for number of acceptable violations"),
```

Comment:
Is it ok to redefine `-l` from upper to lower? I understand the mnemonics at play here...

Another option: keep just one setting, with a colon: `-l 1000:5500`.  We might eventually add lower limits to the other lint programs also, and a more compact notation could be good.

Reply:
Hmm - given that this is a paver command that might be used by external folks, I agree that backwards compatibility would be a plus.
I'll implement this:
For the usual upper limit limit: `-l 5500` (same as always).
For a lower *and* upper limit: `-l 1000:5500` as you suggest.

Reply:
Done now - and tests added.